### ISSUE TITLE
[A11y] Keyboard focus is lost after inserting rows/columns with `editTable(editor, option)` api

### ISSUE THREAD

**Issue by vhuseinova-msft** [microsoft/roosterjs#2731]
**Describe the bug**
Keyboard focus is lost after inserting rows/columns with `editTable(editor, option)` api

**To Reproduce**
1. Open RoosterJS demo app
2. Turn on keyboard navigation or VoiceOver/Narrator
3. Insert a table 
4. Open a table edit menu (from the table) 
![image](https://github.com/microsoft/roosterjs/assets/98852890/69bbf32b-5346-421a-8332-1bcd1b66228b)
5. Select `Insert above` option
6. Notice that keyboard focus is lost

**Expected behavior**
Users should be able to use keyboard navigation after inserting rows/cells. Most of the apps, set selection for the first cell in the inserted range.

**Device Information**
 - OS: [e.g. iOS] MacOS, Windows
 - Browser [e.g. chrome, safari] Edge

**Additional context**

**Comment by vhuseinova-msft** [comment]
Closing this as original ally bug is closed
